### DISCUSSION TITLE
ensemble_boxes

### DISCUSSION THREAD

**Discussion by kwakjeeyoon** [boostcampaitech2/object-detection-level2-cv-06#117] [ideas]
ensemble (universenet, deformable_detr, swin - LB 상위 3개 모델) : default 옵션으로 LB 0.594 나왔습니다! 
LB 점수는 떨어졌는데 다양하게 옵션을 주어 실험해보겠습니다! 

https://github.com/ZFTurbo/Weighted-Boxes-Fusion

**Comment by JiyouSeo** [comment]
효과적이었던 옵션을 정리해 주셔도 괜찮을 것 같습니다~

**Reply by kwakjeeyoon** [reply]
각 기법마다 저자가 적어놓은 최적 옵션대로 먼저 진행해볼 생각입니다. 해당 옵션은 아래와 같습니다!

method | mAP | option
NMS | 0.5642 | IOU Thr: 0.5 | 
Soft-NMS | 0.5616 | Sigma: 0.1, Confidence Thr: 0.001 | 
NMW | 0.5667 | IOU Thr: 0.5 | 
WBF | 0.5982 | IOU Thr: 0.6

=> 옵션으로 돌려본 결과는 슬랙에 공유하겠습니다!

**Comment by jaehwan-AI** [comment]
옵션은 어떻게 다양하게 할 수 있나요?

**Reply by kwakjeeyoon** [reply]
제가 찾은 옵션은 아래와 같습니다! 

option | default | description
--iou_thr (0.5)  : IoU threshold
--skip_box_thr (0.0001) : 최저 confidence (해당 thr 이하의 bbox 삭제)
--sigma (0.1) : Gaussian sigma (to control the variation around its mean value)
--weight : 모델 별 가중치
